Arguments: 
  C:\Program Files\nodejs\node.exe C:\Users\Jorge benitez\AppData\Roaming\npm\node_modules\yarn\bin\yarn.js add express lodash rxjs

PATH: 
  C:\Users\Jorge benitez\bin;C:\Program Files\Git\mingw32\bin;C:\Program Files\Git\usr\local\bin;C:\Program Files\Git\usr\bin;C:\Program Files\Git\usr\bin;C:\Program Files\Git\mingw32\bin;C:\Program Files\Git\usr\bin;C:\Users\Jorge benitez\bin;C:\ProgramData\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0;C:\ProgramData\chocolatey\bin;C:\Program Files\nodejs;C:\Program Files\Java\jdk1.8.0_131\bin;C:\Android\android-sdk\tools;C:\Android\android-sdk\platform-tools;C:\Program Files\Git\cmd;C:\Users\Jorge benitez\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Microsoft VS Code\bin;C:\Users\Jorge benitez\AppData\Roaming\npm;C:\Program Files\Git\usr\bin\vendor_perl;C:\Program Files\Git\usr\bin\core_perl

Yarn version: 
  0.24.5

Node version: 
  8.0.0

Platform: 
  win32 ia32

npm manifest: 
  {
    "name": "rxpress",
    "version": "1.0.0",
    "description": "a rxjs wrapper for express",
    "main": "index.js",
    "scripts": {
      "test": "echo \"Error: no test specified\" && exit 1"
    },
    "keywords": [
      "rx",
      "rxjs",
      "express",
      "reactive",
      "server",
      "http",
      "request"
    ],
    "author": "Andrew Ramos",
    "license": "ISC",
    "devDependencies": {
      "@types/lodash": "^4.14.65",
      "typescript": "^2.3.4"
    }
  }

yarn manifest: 
  No manifest

Lockfile: 
  # THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
  # yarn lockfile v1
  
  
  "@types/lodash@^4.14.65":
    version "4.14.65"
    resolved "https://registry.yarnpkg.com/@types/lodash/-/lodash-4.14.65.tgz#a0f78d71ffcd3c02628d5f616410c98c424326d5"
  
  typescript@^2.3.4:
    version "2.3.4"
    resolved "https://registry.yarnpkg.com/typescript/-/typescript-2.3.4.tgz#3d38321828231e434f287514959c37a82b629f42"

Trace: 
  Error: connect ETIMEDOUT 104.16.59.173:443
      at Object.exports._errnoException (util.js:1026:11)
      at exports._exceptionWithHostPort (util.js:1049:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1136:14)
